kickoff/model.py

```diff
@@ -462,3 +463,88 @@ def getEnUSPlatforms(cls, name):
         releaseTable = getReleaseTable(name.split('-')[0].title())
         release = releaseTable.query.filter_by(name=name).first()
         return json.loads(release.enUSPlatforms)
+
+def getOrderByList(obj, orderByDict = {}):
+    lst = []
+
+    for k, v in orderByDict.iteritems():
+        column = getattr(obj, k)
+        direction = getattr(column, v)
+        lst.append(direction())
+
+    return lst
+
+def getReleasesListView(orderByDict = {}, ready=None, complete=None, status=None, productFilter=None, versionFilter=None,
```

Comment:
You're right, I kept the logic, because, I did not know if any other place (or other system) use getReleases() function. I can change getReleases() function to use only the new View ?